qiskit/transpiler/target.py

```diff
@@ -898,7 +898,7 @@ def build_coupling_map(self, two_q_gate=None):
 
         if two_q_gate is not None:
             coupling_graph = rx.PyDiGraph(multigraph=False)
-            coupling_graph.add_nodes_from(list(None for _ in range(self.num_qubits)))
+            coupling_graph.add_nodes_from([None for _ in range(self.num_qubits)])
```

Comment:
```suggestion
            coupling_graph.add_nodes_from([None] * self.num_qubits)
```
In a particular test, this is 8 times faster.

Reply:
Thats a nice speedup. Similar patterns seem to show up ~100 times around the codebase. Can you make a PR to catch all the relevant cases?

Reply:
That's probably very hard to do without lots of false positives/negative because even innocuous looking constructs like a.b might execute code via properties/descriptors etc would be very context dependent whether you want that code executed for each item or not